### DISCUSSION TITLE
Apartment v1

### DISCUSSION THREAD

**Discussion by glitcheking** [nzp-team/nzportable#474] [Map Releases]
This is one of my firts ever maps from years ago
Its cald APARTMENT 
ITS AL INDOORS SO BE AWARE YOU MIGHT GET KILLED FAST 

THIS MAP FEATURES.
PAP 
8 WALLBUY GUNS
MYSTERY BOX
ALL PERKS 

AND MORE TO COME I AM PLANNING TO MAKE IT INTO A BIGGER MAP WITH EASTER EGGS AND CUSTOM STUFF 

![Nazi_Zombies__Portable_21_11_2023_06_56_13](https://github.com/nzp-team/nzportable/assets/30326405/bb7ea725-b2a8-448a-b329-55052888308e)

![Nazi_Zombies__Portable_21_11_2023_06_54_38_1](https://github.com/nzp-team/nzportable/assets/30326405/3882a064-0041-4d8b-aceb-cb143c6b2b84)

![Nazi_Zombies__Portable_21_11_2023_06_55_26](https://github.com/nzp-team/nzportable/assets/30326405/40978f32-52cb-4b48-875f-169a224446e5)

![Nazi_Zombies__Portable_21_11_2023_06_56_32](https://github.com/nzp-team/nzportable/assets/30326405/2bd276be-4b75-4f4f-86de-569f704a022f)

![Nazi_Zombies__Portable_21_11_2023_06_56_41](https://github.com/nzp-team/nzportable/assets/30326405/6100d42f-48bb-4fdf-9515-44e2152ffc86)

![Nazi_Zombies__Portable_21_11_2023_06_57_37](https://github.com/nzp-team/nzportable/assets/30326405/22306744-ec08-4c0d-ab3e-f6ba84c761b6)

![Nazi_Zombies__Portable_21_11_2023_06_56_25](https://github.com/nzp-team/nzportable/assets/30326405/c385dcc2-bb5a-4a46-b414-097bb159ae06)

![Nazi_Zombies__Portable_21_11_2023_06_54_50](https://github.com/nzp-team/nzportable/assets/30326405/2f1a1fb1-b4e0-429b-b33b-e57ee4869185)

DOWNLOAD---->[apartment.zip](https://github.com/nzp-team/nzportable/files/13427875/apartment.zip)<----

**Comment by MotoLegacy** [comment]
Hi there. In order for this to meet release standards, edit the post to make the download button bigger, and clearer. Otherwise LGTM.
